src/extension.ts

```diff
@@ -271,18 +302,25 @@ function getNonce() {
 	return text;
 }
 
+function makeFrame()
+{
+    const activeEditor = vscode.window.activeTextEditor;
+    if (activeEditor) {
+        activeEditor.selection.active.line;
+    }
+}
+
 const gitExtension = vscode.extensions.getExtension<GitExtension>('vscode.git')
 let git
-console.log("gitEXTPRE", gitExtension?.isActive)
 gitExtension?.activate()
```

Comment:
So we're moving away from using the VSCode git extension and deciding to spawn a child process?
Are we sure we have all permissions?
Will it work across architectures?

Reply:
I don't know the answer to the question, but it's the best I could find. What else do you suggest?

Reply:
VSCode includes an extension to handle all things git. I use it a lot (no more console!).
It's called in the line `vscode.extensions.getExtension<GitExtension>('vscode.git')`
It allows using it's full API.

Reply:
Is it straightforward? Genuine question. 

If it's not, I think we should release this weekend with this approach and later evolve towards your approach.

Reply:
It is _safer_. With less dependencies comes easier adoption.
We could even run the extension on the web (replit or github.dev)